### ISSUE TITLE
Map tiles render on top of the rive models  

### ISSUE THREAD

**Issue by igorskugar** [defold/extension-rive#35]
The bug occurs when using render script found in this repo, located [here](https://github.com/defold/extension-rive/blob/main/defold-rive/lua/rive.render_script).
I've managed to fix the issue by copying the render script to my project and changing the order of `render.draw()` calls.
There are two items on screenshots: health bar made with rive and tilemap.

Before:
```lua
    -- inside update() function
    render.enable_state(render.STATE_STENCIL_TEST)
    render.draw(self.rive_pred)
    render.disable_state(render.STATE_STENCIL_TEST)

    render.draw(self.tile_pred)
    render.draw(self.particle_pred)
    render.draw_debug3d()
```
![image](https://user-images.githubusercontent.com/5592468/138335569-9f71679f-9614-4bf7-b546-67cb28865ddf.png)

After:
```lua
    -- inside update() function
    render.draw(self.tile_pred) -- moved this up

    render.enable_state(render.STATE_STENCIL_TEST)
    render.draw(self.rive_pred)
    render.disable_state(render.STATE_STENCIL_TEST)

    render.draw(self.particle_pred)
    render.draw_debug3d()
```
![image](https://user-images.githubusercontent.com/5592468/138334511-985f2bcb-8627-4e78-a932-22398cb193e1.png)

I don't know if this is the right way to fix the issue, so I'm not creating a PR.

**Comment by dwnu** [comment]
Do you have any new information about this issue?

**Comment by dwnu** [comment]
I see, thanks for update

**Comment by JCash** [comment]
Fixed in [2.0.0](https://github.com/defold/extension-rive/releases/tag/2.0.0)
See https://forum.defold.com/t/updated-rive-extension/75169
